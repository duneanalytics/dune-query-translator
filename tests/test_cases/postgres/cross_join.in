select x, dt from x cross join generate_series(min_dt, current_date, '1 day'::interval) dt
