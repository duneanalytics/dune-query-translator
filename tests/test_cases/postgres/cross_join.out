SELECT
  x,
  dt
FROM x
CROSS JOIN UNNEST(SEQUENCE(min_dt, CURRENT_DATE, TRY_CAST('1 day' AS INTERVAL))) AS _u(dt)